packages/sns/src/governance.canister.ts

```diff
@@ -102,6 +102,29 @@ export class SnsGovernanceCanister extends Canister<SnsGovernanceService> {
     this.assertManageNeuronError(response);
   };
 
+  /**
+   * Remove permissions to a neuron for a specific principal
+   */
+  removeNeuronPermissions = async ({
+    neuronId,
+    permissions,
+    principal,
+  }: SnsNeuronPermissionsParams): Promise<void> => {
+    const request: ManageNeuron = {
```

Comment:
nth: extract a function `toCommand` or something - execpt the `command` I guess the `ManageNeuron` is the same for both add and remove?

Reply:
actually, maybe even refactor following

```
const response = await this.manageNeuron(request);
    this.assertManageNeuronError(response);
```

that's the same too I guess, so you can have a function that takes the `command`  as parameter

Reply:
ok.

Reply:
done